docs/guide/context-variables.md

```diff
@@ -0,0 +1,37 @@
+# Context variables
+
+dbt makes available certain variables for developers to use to control a model's behavior. These variables may be defined within a certain model scope within the model hierarchy or may be global.
+
+## this
+
+`this` returns the schema and table name of the current model. This is useful in any context in which you need to write code that references the current model, for example when defining a `sql_where` clause for an incremental model and for writing pre- and post-model hooks that operate on the model in some way. Developers have options for how to use `this`:
+
+```Python
+this            #returns "schema"."table"
+this.schema     #returns schema
+this.table      #returns temporary table name (dbt creates temp tables before dropping the originals)
```

Comment:
One nitpick -- `this.table` isn't always a temporary name (in the case of incremental models, it's the regular model name). `this.table` is _always_ the same as the name of the table/view that is created/inserted into in the SQL transaction.

If you're doing operations on a table, you want to use `this.table` (or probably just `this`). If you're writing to an audit table (or something else for human consumption), you should use `this.name` which is always the "final" name